src/app/UI/lib/managers/auth/index.js

```diff
@@ -72,12 +72,19 @@ class Auth {
 
         const client = ServiceComBus.instance.getClient("userrepo");
         // Get user with email or _id
-        const users = await client.list("user", {
+        let users = await client.list("user", {
             $or: [
                 { _id: emailOrId },
                 { email: emailOrId }
             ]
         });
+        if (users.length === 0) {
+            // If no user if found try to create which will check against other backends for the user
+            // for example an activeDirectory and if password is ok add user to userrepo backend
+            //users = await client.create("user", {_id: emailOrId, name: emailOrId, backend: "adtest"});
+            users = new Array(await client.create("user", {_id: emailOrId, name: emailOrId, password: password}));
```

Comment:
new Array should never be used. It should work with the returncode from await client.create and if not it should be handled differently than new Array